pkg/dir.props

```diff
@@ -2,6 +2,10 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
 
+  <PropertyGroup>
+    <BuildingAnOfficialBuildLeg Condition="'$(OfficialBuildId)' != '' AND '$(DotNetBuildOffline)' != 'true'">true</BuildingAnOfficialBuildLeg>
```

Comment:
Might want to add `BuildingAnOfficialBuildLeg == ''` here so this can be overridden if needed.

Reply:
Fixed.